unicodetools/src/main/java/org/unicode/text/UCD/MakeUnicodeFiles.java

```diff
@@ -53,7 +59,64 @@ public class MakeUnicodeFiles {
     static boolean DEBUG = false;
 
     public static void main(String[] args) throws IOException {
+
+        boolean clean = Arrays.asList(args).contains("-c");
```

Comment:
How about a more self-explanatory option string, something like "--cleanBin" ?

Reply:
ping -- longer option names make reading and documenting command lines much easier